setup.py

```diff
@@ -53,24 +45,26 @@ def is_platform_mac():
 }
 
 
+# The import of Extension must be after the import of Cython, otherwise
+# we do not get the appropriately patched class.
```

Comment:
this seems odd?

Reply:
Found the appropriate reference in the cython [docs](https://cython.readthedocs.io/en/latest/src/reference/compilation.html):


> Note that when using setuptools, you should import it before Cython as setuptools may replace the Extension class in distutils. Otherwise, both might disagree about the class to use here.

Reply:
that is so weird, but ok